### PR TITLE
Add new device K550 SW-R8BA024

### PR DESCRIPTION
**Pull request by farid1991**

### CODE REVIEW

File: ElfPack/xcl/K550_R8BA024.xcl [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+-Z(CODE)PATCH_KEYHANDLER4=4531B658-4531B65F
+
+-Z(CODE)PATCH_KEYHANDLER1_STACK1=4531B4BA
+-Z(CODE)PATCH_KEYHANDLER1_STACK2=4531B69A
+-Z(CODE)PATCH_KEYHANDLER1_CHANGE1=4531B5F6
+-Z(CODE)PATCH_KEYHANDLER1_CHANGE2=4531B632
+-Z(CODE)PATCH_KEYHANDLER1_CHANGE3=4531B664
+-Z(CODE)PATCH_KEYHANDLER1_CHANGE4=4531B694
+
+// ���������� elf
```
Comment (justdanpo) [inline ElfPack/xcl/K550_R8BA024.xcl]: Could you replace comments with those from other xcl files?

**Review by justdanpo — CHANGES_REQUESTED** [review]

File: ElfPack/xcl/K550_R8BA024.xcl (hunk 1 above)
Comment (farid1991) [reply ElfPack/xcl/K550_R8BA024.xcl]: Oh my fault, will change that tonight. Thank you

Comment (farid1991) [reply ElfPack/xcl/K550_R8BA024.xcl]: Done

Comment (farid1991) [reply ElfPack/xcl/K550_R8BA024.xcl]: Done

**Pull request merged**